src/epub/content.opf

```diff
@@ -80,6 +82,10 @@
 		<meta property="se:url.homepage" refines="#transcriber-3">https://pgdp.net</meta>
 		<meta property="se:url.encyclopedia.wikipedia" refines="#transcriber-3">https://en.wikipedia.org/wiki/Distributed_Proofreaders</meta>
 		<meta property="role" refines="#transcriber-3" scheme="marc:relators">trc</meta>
+		<dc:contributor id="transcriber-4">Robert P. Helms</dc:contributor>
+		<meta property="file-as" refines="#transcriber-4">Helms, Robert P.</meta>
+		<meta property="role" refines="#transcriber-4" scheme="marc:relators">trc</meta>
+		<meta property="se:url.homepage" refines="#transcriber-4">https://www.deadanarchists.org</meta>
```

Comment:
The homepage for producers/contributors should be a *personal* homepage - this doesn't look like that. Can you remove this?

Reply:
Ok, I fixed this in a rebase.